src/dawn/engine_dawn.cc

```diff
@@ -317,7 +281,7 @@ MapResult MapTextureToHostBuffer(const RenderPipelineInfo& render_pipeline,
     }
   }
   // Always unmap the buffer at the end of the engine's command.
-  fb_buffer.Unmap();
+  render_pipeline.fb_buffer.Unmap();
 
```

Comment:
Also need to copy the buffer data for buffers that are not colour-attachments back into the Amber buffer objects.
Need a TODO here at least.